plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -1025,15 +1025,19 @@ def send_signed_transaction(self, tx_signed: JSONLike) -> Optional[str]:
         :param tx_signed: the signed transaction
         :return: tx_digest, if present
         """
-        tx_digest = self._try_send_signed_transaction(tx_signed)
+        tx_digest = self._try_send_signed_transaction(tx_signed, raise_on_try=True)
```

Comment:
raise_on_try should be a kwarg of send_signed_transaction and default to False. Then in the ledger connection we can pass the appropriate keyword

Reply:
Done in 8d0f35b107d57b7674129f31ce47b1f85e67ff9d